general-concepts/mirrors/text.xml

```diff
@@ -91,8 +98,14 @@ requirement of the license.
 
 <p>
 When you upload the file to <c>dev.gentoo.org:~/public_html</c>, you must ensure that it, and its
-parent directories, are world-readable.
+parent directories, are world-readable.  An example <c>SRC_URI</c> referencing
+a distfile mirrored this way follows:
 </p>
+
+<codesample lang="ebuild">
+SRC_URI="https://dev.gentoo.org/~myname/${P}.tar.gz"
```

Comment:
I would suggest a better example, like
```
<codesample lang="ebuild">
SRC_URI="https://dev.gentoo.org/~myname/distfiles/${PN}/${P}.tar.gz"
</codesample>
```
Because we don't want to end up re-creating the same problem /distfiles have (single folder containing too many files). It will also help developer to cleanup files because files are logically grouped.